### PR TITLE
Issue-156 (Chore)[Bug]: Fixed Broken Links

### LINKED ISSUE #156: Bug🐛: Fix 6 Broken Links
### Describe the bug.

Fix the first six broken links (all 404s).  The page may exist and have been re-named, or the page may not exist.  Determine the situation and fix the link, or possibly remove the link entirely if it cannot be fixed.

For the last two, it will require modifying the GHA to enter an exception, so it's out of scope for this issue.

### Describe the steps to reproduce the behavior

1. Go to this page: https://github.com/open-horizon/open-horizon.github.io/actions/runs/1866277862
2. Click on the `Build` button
3. Click on the `>` beside "Run check for Broken Link"
4. You will see a list of 8 "broken" links.

### Expected behavior.

_No response_

### Screenshots.

_No response_

### Operating environment

n/a

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Abhijay007**
Signed-off-by: Abhijay Jain [Abhijay007j@gmail.com](mailto:Abhijay007j@gmail.com)

## Description

Fixed 6 Broken links 

Fixes #156 

## Type of change

code

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

locally by running server 

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked my code and corrected any misspellings
- [x] I have tagged the reviewers in a comment below incase my pull request is ready for a review
- [x] I have signed the commit message to agree to Developer Certificate of Origin (DCO) (to certify that you wrote or otherwise have the right to submit your contribution to the project.) by adding "--signoff" to my git commit command.

### CODE REVIEW

**Comment by Abhijay007** [comment]
@joewxboy for the links "[accessing_ui.md](https://open-horizon.github.io/docs/console/accessing_ui.md) and [cloudctl_oc_cli.md](https://open-horizon.github.io/docs/cli/cloudctl_oc_cli.md) the console folder does not exist in docs in the English language so I added links in `es`  lang for now, If required I will covert those files in English and add links to them.

File: docs/installing/preparing_edge_cluster.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Install one of these edge clusters and prepare it for the {{site.data.keyword.ed
 
 1. Install OCP by following the installation instructions in the [{{site.data.keyword.open_shift_cp}} Documentation ](https://docs.openshift.com/container-platform/4.5/welcome/index.html){:target="_blank"}{: .externalLink}. ({{site.data.keyword.ieam}} only supports OCP on x86_64 platforms.)
 
-2. Install the Kubernetes CLI (**kubectl**), Openshift client CLI (**oc**) and Docker on the admin host where you administer your OCP edge cluster. This is the same host where you run the agent installation script. For more information, see [Installing cloudctl, kubectl, and oc](../cli/cloudctl_oc_cli.md).
+2. Install the Kubernetes CLI (**kubectl**), Openshift client CLI (**oc**) and Docker on the admin host where you administer your OCP edge cluster. This is the same host where you run the agent installation script. For more information, see [Installing cloudctl, kubectl, and oc](../nl/es/cli/cloudctl_oc_cli.md).
```
Comment (joewxboy) [inline docs/installing/preparing_edge_cluster.md:34]: The cloudctl_oc_cli document was contributed by IBM and does not appear to apply to Open Horizon.  That's why the file does not exist in US English for Open Horizon.  We should unlink this file unless and until we create an Open Horizon-specific version of the page.

File: docs/user_management/security_privacy.md [hunk 2]
```diff
@@ -62,7 +62,7 @@ A compromised agbot can attempt to propose malicious workload deployments, but t
...
...cases where it is required. In {{site.data.keyword.ieam}} you have the ability to specify that an application service should be deployed with privileged process execution enabled. By default, it is disabled. You must explicitly enable it in the [deployment configuration](https://github.com/open-horizon/anax/blob/master/docs/deployment_string.md) of the respective Service Definition file for each service that needs to run in  this mode. And further, any node on which you want to deploy that service must also explicitly allow privileged mode containers. This ensures that node owners have some control over which services are executing on their edge nodes. For an example of how to enable privileged mode policy on an edge node, see [privileged node policy](https://github.com/open-horizon/anax/blob/master/cli/samples/privileged_node_policy.json). If the service definition or one of its dependencies requires privileged mode, the node policy must also allow privileged mode, or else none of the services will not be deployed to the node. For an indepth discussion of privileged mode see [What is privileged mode and do I need it?](https://wiki.lfedge.org/pages/viewpage.action?pageId=44171856).
```
Comment (joewxboy) [inline docs/user_management/security_privacy.md:65]: It looks like we're missing the code for external links on the wiki.lfedge.org hyperlink.  Can you please add it here?

File: docs/using_edge_services/service_rollbacks.md [hunk 3]
```diff
@@ -131,4 +131,4 @@ hzn eventlog list
 ```
 {: codeblock}
 
-Lastly, you can also use the [management console](../console/accessing_ui.md) to modify rollback deployment versions settings. You can do this when creating a new deployment policy or by viewing and editing existing policy details including rollbacks settings. Note the term “time frame” in the rollback section of the UI is equivalent to “retry_durations” in the CLI.
+Lastly, you can also use the [management console](../nl/es/console/accessing_ui.md) to modify rollback deployment versions settings. You can do this when creating a new deployment policy or by viewing and editing existing policy details including rollbacks settings. Note the term “time frame” in the rollback section of the UI is equivalent to “retry_durations” in the CLI.
```
Comment (joewxboy) [inline docs/using_edge_services/service_rollbacks.md:134]: Open Horizon does not have a UI.  We should remove the whole "Lastly" paragraph as it does not apply.

**Review by joewxboy — CHANGES_REQUESTED** [review]
There are some changes that should be made.  See the inline comments.

File: docs/installing/preparing_edge_cluster.md (hunk 1 above)
Comment (Abhijay007) [reply docs/installing/preparing_edge_cluster.md:34]: so do I need to add an empty link or remove that para in order to unlink it?

File: docs/user_management/security_privacy.md (hunk 2 above)
Comment (Abhijay007) [reply docs/user_management/security_privacy.md:65]: sure :)

File: docs/using_edge_services/service_rollbacks.md (hunk 3 above)
Comment (Abhijay007) [reply docs/using_edge_services/service_rollbacks.md:134]: ok

File: docs/installing/preparing_edge_cluster.md (hunk 1 above)
Comment (joewxboy) [reply docs/installing/preparing_edge_cluster.md:34]: How about removing the "For more information" sentence for now.  Can you open a separate issue identifying this page (docs/installing/preparing_edge_cluster.md) as a candidate for re-writing?

Comment (Abhijay007) [reply docs/installing/preparing_edge_cluster.md:34]: sure, will do that :)

Comment (Abhijay007) [reply docs/installing/preparing_edge_cluster.md]: I am opening an issue for the re-writing the page  (docs/installing/preparing_edge_cluster.md) so can you please provide me more details about it, like what changes to be specific we need?

**Review by joewxboy — APPROVED** [review]

**Comment by joewxboy** [comment]
@Abhijay007 Looks like your commit wasn't signed.  Can you amend the commit to add the sign-off flag?

**Review by joewxboy — APPROVED** [review]

**Pull request merged**